### ISSUE TITLE
Pressing Control+D exits mathicsscript leaving the mathics process running (in Arch Linux, Xfree4 and KDE Konsole)

### ISSUE THREAD

**Issue by DUOLabs333** [Mathics3/mathicsscript#57]
Pressing Control+D does not kill mathicsscript. Arch Linux. `Quit[]` does work though.

**Comment by DUOLabs333** [comment]
It exits, but the process is still running in the background.

**Comment by rocky** [comment]
Two things. First of all, I am not seeing this behavior. Second, if this is going on this wasn't communicated very well. What does this have to do with Control-D and needs to be contrasted or compared to  `Quit[]` ?

**Comment by DUOLabs333** [comment]
It says when running mathicsscript, that Control+D should quit it. It does not (or at least not properly). `Quit[]` quits properly.

**Comment by TiagoCavalcante** [comment]
@DUOLabs333 see [this](https://bbs.archlinux.org/viewtopic.php?id=225225). As some UX searches discovered, having too many settings is bad, and biding a key combination to an Expression seems the type of thing that won't work well, and won't work in all systems.

The thing that seems right is to kill the process when an EOF is received (which seems to be the current behavior).

**Comment by DUOLabs333** [comment]
I'm using Konsole from KDE.

**Comment by DUOLabs333** [comment]
@TiagoCavalcante However, your comment about the terminal I was using caused me into looking in the Keyboard settings -- apparently I was on `Xfree 4`. Switching to `Linux console` fixes my problem

**Comment by DUOLabs333** [comment]
dvddfgvd dfedg

**Comment by DUOLabs333** [comment]
This no longer works in `4.1.0.dev0`.

**Comment by rocky** [comment]
See if you can [git bisect](https://git-scm.com/docs/git-bisect) to find the commit where this went bad. I am not convinced though this isn't a local environment problem.

**Comment by DUOLabs333** [comment]
You're right -- it works with xterm, so it's something with Konsole. I'll look more into this.
